src/Magento/FunctionalTestingFramework/Test/Util/TestObjectExtractor.php

```diff
@@ -155,4 +170,34 @@ public function extractTestData($testData)
             throw new XmlException($exception->getMessage() . ' in Test ' . $filename);
         }
     }
+
+    /**
+     * Append names of test files, including merge files, in description annotation
+     *
+     * @param string $description
+     * @param array  $fileNames
+     *
+     * @return string
+     */
+    private function appendFileNamesInDescriptionAnnotation($description, $fileNames)
+    {
+        $title = '';
+        foreach ($fileNames as $fileName) {
+            if (!empty($fileName && realpath($fileName) !== false)) {
```

Comment:
Line should read 
```
if (!empty($fileName) && realpath($fileName) !== false) {
```
Note `empty()` function closing parenthesis

Reply:
Fixed. I wonder how it even worked?